packages/node_modules/@ciscospark/plugin-people/test/integration/spec/people.js

```diff
@@ -73,6 +77,33 @@ describe('plugin-people', function () {
           assert.equal(peopleResponse[1].emails[0], spock.email);
         }));
 
+      // list([VALID_ID, VALID_ID]) -> valid response with 2 users, nothing in notFoundIds
+      it('gets a group of found people by ids', () => spock.spark.people.list([mccoy.id, spock.id, bones.id])
+        .then((peopleResponse) => {
+          assert.equal(peopleResponse.length, 3);
+          assert.isPerson(peopleResponse[0]);
+          assert.isPerson(peopleResponse[1]);
+          assert.isPerson(peopleResponse[2]);
+          assert.equal(peopleResponse[0].emails[0], mccoy.email);
+          assert.equal(peopleResponse[1].emails[0], spock.email);
+          assert.equal(peopleResponse[2].emails[0], bones.email);
+        }));
+      // list([VALID_ID, INVALID_ID]) -> valid response with VALID_ID user, and INVALID_ID in notFoundIds
```

Comment:
(nitpick) can we get an empty space between `it` blocks?

Reply:
+1